server.py

```diff
@@ -18,6 +19,8 @@
     app.config['SQLALCHEMY_DATABASE_URI'] = 'mysql+pymysql://root:@localhost/payments'
```

Comment:
This if/else branch should be removed since the logic has been moved into get_database_uri().